GameServerLib/Logic/Game.cs

```diff
@@ -58,8 +58,8 @@ Logger logger
         public void Initialize(Address address, string blowfishKey)
         {
             _logger.LogCoreInfo("Loading Config.");
-
-            Config = new Config(Program.ConfigPath);
+            
+            Config = new Config(_logger, Program.ConfigPath, Program.ConfigJSON);
```

Comment:
I don't think we want to be passing the config path AND the config JSON around this much. This probably should be reworked to read the config file already in `GameServerApp`, and create the config object already there. I'll add more detailed instructions in another comment.